frontend/src/metabase/dashboard/components/DashCard/DashCardTabMenu/DashCardTabMenu.tsx

```diff
@@ -23,6 +27,22 @@ export function DashCardTabMenu({ dashCardId }: DashCardTabMenuProps) {
     return tabs.filter(t => t.id !== selectedTabId);
   }, [selectedTabId, tabs]);
 
+  const moveDashcard = useCallback(
+    (destTabId: number) => {
+      dispatch(moveDashCardToTab({ dashCardId, destTabId }));
+      dispatch(
+        addUndo({
+          message: t`Card moved to ${tabs.find(c => c.id === destTabId)?.name}`,
```

Comment:
This is also not apparent. I'm not sure what `c` is supposed to mean, is it for Card? But it's finding on the `tabs`. Should this be
```suggestion
          message: t`Card moved to ${tabs.find(tab => tab.id === destTabId)?.name}`,
```
?